test/Directory.Build.props

```diff
@@ -6,7 +6,7 @@
     <TargetFramework Condition="$(Configuration.EndsWith('-netfx'))">net461</TargetFramework>
     <TargetFramework Condition="'$(UseIntrinsics)' == 'true'">netcoreapp3.0</TargetFramework>
     <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net461'">win-x64</RuntimeIdentifier>
-    <DefineConstants Condition="'$(TargetFramework)' == 'netcoreapp3.0'">NETCOREAPP30</DefineConstants>
+    <AllowedReferenceRelatedFileExtensions>$(AllowedReferenceRelatedFileExtensions);.runtimeconfig.json;.runtimeconfig.dev.json;.deps.json</AllowedReferenceRelatedFileExtensions>  
```

Comment:
This is required in the projects that are referencing the required project

Reply:
That would make a fine comment in the file.